File: HotReload/HotReload/ViewController.m
//
//  ViewController.m
//  HotReload
//
//  Created by zhoufei on 2022/4/29.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
    self.titleLabel.text = @"我是热重载";
}

- (void)injected {
    self.titleLabel.text = @"我是热重载2r  s";
}


@end

File: Swift-OC/Swift-OC/ViewController.swift
//
//  ViewController.swift
//  Swift-OC
//
//  Created by zhoufei on 2023/3/30.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        
        let p = Person();
        p.name = "jack";
        p.age = 11;
        
        p.myCar = Car(speed: 101.1, name: "宝马")
        
        p.say();
        p.play();
    }


}

